<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVExportImportDialog</class>
 <widget class="QDialog" name="CSVExportImportDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>288</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>&lt;WindowTitle&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Field Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The character that is used to separate columns (default: comma)</string>
        </property>
        <property name="text">
         <string>Separator:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="text_separator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>,</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
        <property name="placeholderText">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="check_enclosed_by">
        <property name="toolTip">
         <string>Allow fields to optionally be enclosed by this character; this allows the separator to be used within a field (default: &quot;)</string>
        </property>
        <property name="text">
         <string>Enclosed by:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="text_enclosed_by">
        <property name="text">
         <string>&quot;</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
        <property name="placeholderText">
         <string>&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="check_multi_line">
        <property name="toolTip">
         <string>Whether enclosed fields can span multiple lines </string>
        </property>
        <property name="text">
         <string>Allow multi-line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_short_lines">
        <property name="toolTip">
         <string>Allow different rows to contain a different number of columns; not setting this will cause the import to error if this is the case.</string>
        </property>
        <property name="text">
         <string>Allow short lines</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_header_line">
        <property name="toolTip">
         <string>Whether the first line should be treated as a header; when exporting: whether to include the first line as a header</string>
        </property>
        <property name="text">
         <string>First line is header</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_enclosed_by</sender>
   <signal>toggled(bool)</signal>
   <receiver>text_enclosed_by</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
